1_getHeadlines.js

```diff
@@ -78,8 +78,9 @@ function getNextHeadline() {
 	}).on('error', error.bind(this, searchTerm));
 }
 
-http.get(config.frontPageURL, response => {
+https.get(config.frontPageURL, response => {
   let running = false;
+  console.info(`Fetching startpage. Status code ${response.statusCode}`);
```

Comment:
Your indentation is a bit weird :)

Reply:
Whoops, looks like I made a first step into the tabs+spaces mix hell. Time to ask autoformat for help, I guess …